### PR TITLE
Added login + logout functionality between pages

### PR DESCRIPTION
**Pull request by hannahsalameh**
(Basically splits loginScript.js into 2) Adds calls to the login and logout functions as well. 

Also, moved web.xml to its correct location as well as the images folder.

### CODE REVIEW

File: src/main/webapp/loggedin/logout.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+document.addEventListener('DOMContentLoaded', (event) => {
+    var usernameElem = document.getElementById("username");
+    if(usernameElem){
+        usernameElem.innerHTML = localStorage.getItem("username");
+    }
+});
+ 
+//signs out of the page
+function signOut() {
+    //log out page --> DOESNT RUN FOR SOME REASON??
```
Comment (devonwasson) [inline src/main/webapp/loggedin/logout.js:11]: is this comment still relevant?

File: src/main/webapp/loggedin/logout.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+ 
+    //log out google
+  gapi.load('auth2', function() {
+       gapi.auth2.init({
+         client_id: '610251294652-9ojdhjhh8kpcdvdbmrvp9nkevgr2n2d8.apps.googleusercontent.com'
```
Comment (devonwasson) [inline src/main/webapp/loggedin/logout.js:16]: I don't think we want this key to be stored here...

File: src/main/webapp/loginScript.js [hunk 3]
```diff
@@ -1,50 +1,22 @@
...
-    }).then(function(auth2){
-        auth2.signOut();
-        localStorage.removeItem("username");
-        window.location = "login.html";
-    });
-  });
-}
-
-//current method to check sign in
-function checkSignIn(){
```
Comment (devonwasson) [inline src/main/webapp/loginScript.js:40]: don't we still nee the checkSignIn() function?

Comment (hannahsalameh) [reply src/main/webapp/loginScript.js:40]: no, this has been replaced with the filter and session ID method of checking pages

File: src/main/webapp/loggedin/logout.js (hunk 2 above)
Comment (hannahsalameh) [reply src/main/webapp/loggedin/logout.js:16]: oh shoot yea ur right

File: src/main/webapp/loggedin/logout.js [hunk 4]
```diff
@@ -8,12 +8,12 @@ document.addEventListener('DOMContentLoaded', (event) => {
...
 //signs out of the page
 function signOut() {
-    //log out page --> DOESNT RUN FOR SOME REASON??
+    var api_token = getToken();
  
     //log out google
   gapi.load('auth2', function() {
        gapi.auth2.init({
-         client_id: '610251294652-9ojdhjhh8kpcdvdbmrvp9nkevgr2n2d8.apps.googleusercontent.com'
+         client_id: api_token
```
Comment (devonwasson) [inline src/main/webapp/loggedin/logout.js]: I would just combine lines 11 and 16.

**Pull request merged**